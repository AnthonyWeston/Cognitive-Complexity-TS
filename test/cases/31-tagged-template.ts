function noRecursion(arr: TemplateStringsArray, arg: string): string {
    return recursion`${"str"}`;
}

function recursion(arr: TemplateStringsArray, arg: string): string {
    return recursion`${"str"}`;
}
