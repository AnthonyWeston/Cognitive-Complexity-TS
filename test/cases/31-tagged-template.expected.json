{
    "31-tagged-template.ts": {
        "score": 1,
        "inner": [
            {
                "name": "noRecursion",
                "score": 0,
                "line": 1,
                "column": 1
            },
            {
                "name": "recursion",
                "score": 1,
                "line": 5,
                "column": 1
            }
        ]
    }
}